.golangci.yml

```diff
@@ -138,7 +137,7 @@ issues:
   # The list of ids of default excludes to include or disable. By default it's empty.
   # See the list of default excludes here https://golangci-lint.run/usage/configuration.
   include:
-    - EXC0001
+    # - EXC0001 - errcheck checks that are not usually checked
```

Comment:
We discussed this on https://github.com/open-telemetry/opentelemetry-collector/pull/2881#discussion_r616834777